article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+	with k factors on the set of covariates the modeler is interested in. 
+	\item Second, check the factor model's accuracy using posterior predictive 
+	checks. 
+	\item Once a sufficiently accurate latent variable model is recovered, use it 
+	to estimate an expected value of the latent variable for each observation, 
+	and control for this value in the outcome model, alongside the treatment 
+	variables and other covariates of interest. 
+\end{itemize}
+
+
+
+One main assumption of the deconfounder algorithm is that the data at hand 
+should only have multi-cause confounders, thus the title of the paper, "The 
+blessings of multiple causes". In other words, this method works when all 
+unobserved confounders affect multiple of the observed causes (or treatment 
+variables) of interest, alongside the outcome. This assumption is weaker than 
+the one required for ignorability to hold, which requires the absence of both 
+single cause and multi-cause confounders for accurate causal inferences. 
+
+
+
+\subsection{Case study: Simulation}
+\label{sec:deconfounder-simulation}
+
+The purpose of this section is to investigate the effectiveness of the 
+deconfounder algorithm \citep{wang_2019_blessings} in adjusting for unobserved 
+confounding. We use simulated mode choice data where travel distance 
+linearly confounds both travel time and travel cost. We then mask the travel 
+distance data and treat it as an unobserved variable. 
+
+
+We estimate three models:
+
+\begin{itemize}
+	\item Model 1: A multinomial logit with the correct original specification, 
+	except we ommit the travel distance variable in the specification without 
+	trying to adjust for it. This model represents the worst case scenario 
+	where a modeler ignores, or is unaware of, unobserved confounding.
+	\item Model 2: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance). In this method, we use all the variables in 
+	each mode's utility to recover that mode's confounder. This is in line 
+	with the approach taken in \citet{wang_2019_blessings}, where they use all the 
+	of observed variables in the factor model to recover a substitute 
+	confounder.
+	\item Model 3: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance), but this time, we only use travel time and 
+	cost in the factor model, instead of all the variables in the utility 
+	specification of each mode. By only using what we know are confounded 
+	variables to recover the substitute confounder, our goal is to analyze 
+	whether we can improve the accuracy of this approach by adopting a stronger 
+	prior on which variables are confounded. This can 
+	be in the form of building and testing candidate causal graphs that 
+	try to illustrate this confounding. 
+\end{itemize}
+
+
+We compare both the coefficient estimates on travel time and cost from 
+each of those three models to the true estimates used in the simulation. 
```

Comment:
Where is the result. of the comparison? In general I think we should show not tell. We should show a table of the simulated estimation results. For example, True value | 2.5%-tile simulated estimates | Mean | 97.5%-tile simulated estimates. Then in the sentence we can say, "In Table 1, we compare [...]".

Reply:
One row per estimation method.

Reply:
Alternatively, the plot you have in `ho_deconfounder_on_sim_data` would be great, provided you rotate the x-axis-tick-labels and split them all so the parenthetical description is on the second line.

Reply:
See edits in #177.